### DISCUSSION TITLE
Replicating select popup composite behavior

### DISCUSSION THREAD

**Discussion by georgekaran** [ariakit/ariakit#1829] [Questions] [answered]
Currently, I have a component that is very similar to [this](https://codesandbox.io/s/shy-fire-qckpjq?file=/src/select-multiple.tsx).

In this component I cannot use `composite` as `true`, which would imply in all the `Popover` as being the `listbox`, and, since I have some buttons inside the `Popover` using `composite` is not an option.

But at the same time, I'm trying to achieve a very similar behavior:
1. Open the Select
2. The `listbox` is focused
3. Pressing `Tab` would focus on the `button 1`
4. Pressing `Tab` again would focus on the `button 2`
5. Finally, pressing `Tab` would go back to the list as being focused.

How can I achieve this behavior without using `composite` which would hurt `aria` specification?

**Comment by diegohaz** [comment, marked as answer]
Have you tried `modal`?
```diff
  <SelectPopover
-   portal
+   modal
  />
```
